python/pyspark/testing/connectutils.py

```diff
@@ -57,5 +57,8 @@ def __getattr__(self, item: str) -> Any:
 class PlanOnlyTestFixture(unittest.TestCase):
     @classmethod
     def setUpClass(cls: Any) -> None:
-        cls.connect = MockRemoteSession()
+        cls._connect = MockRemoteSession()
         cls.tbl_name = f"tbl{uuid.uuid4()}".replace("-", "")
+
+    def connect(self) -> "MockRemoteSession":
```

Comment:
Should probably a property instead of a method. Let's do it with `connect: RemoteSparkSession` I think that's better.